examples/dashboard-from-configmap/resources.yml

```
@@ -0,0 +1,132 @@
+---
+# We need to extend the permissions of the Grafana serviceaccount to watch for configmaps
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: sidecar-role
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "watch", "list"]
+---
+# We bind the new permissions to the grafana-sa
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sidecar-rolebinding
+roleRef:
+  kind: ClusterRole
+  name: sidecar-role
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: grafana-sa
+    namespace: grafana # TODO: replace with the namespace where Grafana is deployed
+---
+# Provisioning config
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: provisioning-config
```

```suggestion
  name: provisioning-config
  namespace: grafana
```
I think it'd make sense to be explicit about namespace since we already rely on it in `ClusterRoleBinding`.